sovrin_client/test/cli/test_nym.py

```diff
@@ -282,3 +282,42 @@ def testSend2NymsSucceedsWhenBatched(
     do('send GET_NYM dest={dest}',
         mapper=parameters, expect=CURRENT_VERKEY_FOR_NYM, within=2)
 
+def test_send_different_nyms_succeeds_when_batched(
```

Comment:
there are 2 requests, one with "entercmd" and the second with "do".  "entercmd" doesn't send it to the server so when "do" executes it will have both in a batch.  I've entered comments to clarify this.